apps/sms/test/marionette/conversation_test.js

```diff
@@ -16,10 +17,20 @@ marionette('Conversation Panel Tests', function() {
     '/mocks/mock_navigator_moz_mobile_message.js',
     '/mocks/mock_navigator_moz_contacts.js'
   ];
+  var apps = {};
+  apps[MessagesActivityCaller.ORIGIN] = __dirname + '/apps/activitycaller';
 
-  var client = marionette.client();
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'focusmanager.testmode': true
```

Comment:
oh I see you just copied it from elsewhere. Maybe @azasypkin would know better ?